jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.c

```diff
@@ -2507,18 +2513,20 @@ ecma_builtin_array_prototype_object_reduce (ecma_value_t this_arg, /**< this arg
       /* 9. */
       ecma_value_t current_index;
 
-      for (; index < len && ecma_is_value_empty (ret_value); index++)
+      while (index < len && ecma_is_value_empty (ret_value))
```

Comment:
Is it a personal preference to remove the for or something else?

Reply:
It was changed due to the `index` update in the previous versions. With that part reverted, the `for` might be a better choice indeed.